common/cache/lru_test.go

```diff
@@ -384,3 +384,49 @@ func TestPanicOptionsIsNil(t *testing.T) {
 
 	New(nil)
 }
+
+func TestEvictItemsPastTimeToLive_AllExpired_ActivelyEvict(t *testing.T) {
+	cache := New(&Options{
+		MaxCount:      5,
+		TTL:           time.Millisecond * 75,
+		ActivelyEvict: true,
+	})
+
+	_, err := cache.PutIfNotExist("A", t)
+	assert.NoError(t, err)
```

Comment:
this should be required.
My rationale for assert/require - assert only for the path that you test. Everything that is a test setup/error checks should be require, since there is no point of proceeding in this test if PutIfNotExist is broken.